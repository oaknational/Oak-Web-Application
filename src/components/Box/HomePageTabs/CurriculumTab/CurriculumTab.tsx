import { FC } from "react";

import Box from "../../Box";

import Badge from "@/components/Badge/";
import ButtonAsLink from "@/components/Button/ButtonAsLink";
import Flex from "@/components/Flex";
import { GridArea } from "@/components/Grid";
import Grid from "@/components/Grid/Grid";
import Illustration from "@/components/Illustration/Illustration";
import MaxWidth from "@/components/MaxWidth/MaxWidth";
import { Heading, LI, UL } from "@/components/Typography";
import Typography from "@/components/Typography/Typography";

const CurriculumTab: FC = () => {
  return (
    <Flex $background={"aqua"} $pv={24}>
      <MaxWidth $ph={[16]}>
        <Grid $cg={16}>
          <GridArea $colSpan={[12, 6]}>
            <Flex
              $flexDirection={"column"}
              $maxWidth={[640]}
              $pt={32}
              $alignItems={"flex-start"}
              $gap={24}
              $flex={"0 1 auto"}
            >
              <Heading $font={"heading-7"} tag={"h1"}>
                Teachers & subject leads
              </Heading>
              <Heading $font={"heading-3"} tag={"h2"}>
                New curriculum plans
              </Heading>
              <Flex $flexDirection={"column"}>
                {" "}
                <Typography $font={"body-1"}>
                  Explore our new curriculum plans and early-release units.
                </Typography>
                <UL $mt={16} $font={"body-1"}>
                  <LI>National curriculum-aligned</LI>
                  <LI>Primary and secondary sequences</LI>
                  <LI>Designed by curriculum experts</LI>
                </UL>
              </Flex>

              <Flex $gap={24} $flexWrap={"wrap"}>
                <ButtonAsLink
                  label={"View curriculum plans"}
                  variant={"brushNav"}
                  page={"curriculum-landing-page"}
                  viewType={"teachers-2023"}
                  icon={"arrow-right"}
                  $iconPosition={"trailing"}
                  iconBackground="black"
                />
                <ButtonAsLink
                  label={"Early-release units"}
                  variant={"brushNav"}
                  page={"early-release-units-page"}
                  viewType={"teachers-2023"}
                  icon={"arrow-right"}
                  $iconPosition={"trailing"}
                  iconBackground="black"
                />
              </Flex>
            </Flex>
          </GridArea>
          <GridArea $colSpan={[12, 6]} $alignItems={"flex-end"}>
            <Flex
              $pv={64}
              $flexDirection={"column"}
              $justifyContent={"space-between"}
              $alignItems={"flex-end"}
              $flex={"0 1 auto"}
              $position={"relative"}
              $minWidth={[0, 350]}
              $display={["none", "flex"]}
              $maxWidth={524}
            >
              <Flex $flexDirection={"column"} $gap={10}>
                <Illustration
                  slug={"teacher-whiteboard"}
                  noCrop
                  $objectFit="contain"
                  priority
                  $ba={3}
                  $borderStyle={"solid"}
                  $borderColor={"black"}
                />
                <Typography $font={"body-1"} $color={"black"}>
                  50% of teachers feel more confident in curriculum design.
                </Typography>
              </Flex>

              <Box $position={"absolute"} $bottom={120} $left={-24}>
                {" "}
                <Badge icon="bell" />
              </Box>
            </Flex>
          </GridArea>
        </Grid>
      </MaxWidth>
    </Flex>
  );
};

export default CurriculumTab;
