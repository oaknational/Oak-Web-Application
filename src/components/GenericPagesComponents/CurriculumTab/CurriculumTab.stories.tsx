import React from "react";
import { ComponentStory, ComponentMeta } from "@storybook/react";

import Component from "./CurriculumTab";

import AnalyticsDecorator from "@/storybook-decorators/AnalyticsDecorator";

export default {
  decorators: [AnalyticsDecorator],
  component: Component,
  argTypes: {},
} as ComponentMeta<typeof Component>;

const Template: ComponentStory<typeof Component> = (args) => (
  <Component {...args} />
);

export const CurriculumTab = Template.bind({});
