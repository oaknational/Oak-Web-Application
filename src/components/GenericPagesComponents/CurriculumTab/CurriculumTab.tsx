import { FC } from "react";
import {
  OakGrid,
  OakGridArea,
  OakHeading,
  OakLI,
  OakUL,
  OakTypography,
} from "@oaknational/oak-components";

import Box from "@/components/SharedComponents/Box";
import CurriculumTabBadge from "@/components/GenericPagesComponents/CurriculumTabBadge";
import ButtonAsLink from "@/components/SharedComponents/Button/ButtonAsLink";
import Flex from "@/components/SharedComponents/Flex";
import Illustration from "@/components/SharedComponents/Illustration";
import MaxWidth from "@/components/SharedComponents/MaxWidth";

const CurriculumTab: FC = () => {
  return (
    <Flex $background={"aqua"} $pv={24}>
      <MaxWidth $ph={[16]} $pb={24}>
        <OakGrid $cg={"all-spacing-4"}>
          <OakGridArea $colSpan={[12, 6]}>
            <Flex
              $flexDirection={"column"}
              $maxWidth={[640]}
              $pt={32}
              $alignItems={"flex-start"}
              $gap={24}
              $flex={"0 1 auto"}
            >
              <OakHeading $font={"heading-7"} tag={"h1"} $color={"grey70"}>
                Teachers & subject leads
              </OakHeading>
              <OakHeading $font={"heading-3"} tag={"h2"}>
                New curriculum plans
              </OakHeading>
              <Flex $flexDirection={"column"}>
                {" "}
                <OakTypography $font={"body-1"}>
                  Explore our new curriculum plans.
                </OakTypography>
                <OakUL $mt="space-between-s" $font={"body-1"}>
                  <OakLI>National curriculum-aligned</OakLI>
                  <OakLI>Primary and secondary sequences</OakLI>
                  <OakLI>Designed by curriculum experts</OakLI>
                </OakUL>
              </Flex>

              <Flex $gap={24} $flexWrap={"wrap"} $pb={24}>
                <ButtonAsLink
                  label={"View curriculum plans"}
                  variant={"brushNav"}
                  page={"curriculum-landing-page"}
                  icon={"arrow-right"}
                  $iconPosition={"trailing"}
                  iconBackground="black"
                />
              </Flex>
            </Flex>
          </OakGridArea>
          <OakGridArea $colSpan={[12, 6]} $alignItems={"flex-end"}>
            <Flex
              $pv={64}
              $flexDirection={"column"}
              $justifyContent={"space-between"}
              $alignItems={"flex-end"}
              $flex={"0 1 auto"}
              $position={"relative"}
              $minWidth={[0, 350]}
              $display={["none", "flex"]}
              $maxWidth={524}
              $pl={20}
            >
              <Flex $flexDirection={"column"} $gap={10}>
                <Illustration
                  slug={"teacher-whiteboard"}
                  noCrop
                  $objectFit="contain"
                  priority
                  $ba={3}
                  $borderStyle={"solid"}
                  $borderColor={"black"}
                />
                <OakTypography $font={"body-1"} $color={"black"}>
                  50% of teachers feel more confident in curriculum design.
                </OakTypography>
              </Flex>

              <Box
                $position={"absolute"}
                $bottom={120}
                $left={-20}
                aria-hidden={true}
              >
                {" "}
                <CurriculumTabBadge icon="bell" />
              </Box>
            </Flex>
          </OakGridArea>
        </OakGrid>
      </MaxWidth>
    </Flex>
  );
};

export default CurriculumTab;
