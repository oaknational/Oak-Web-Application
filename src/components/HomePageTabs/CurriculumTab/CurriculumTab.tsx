import { FC } from "react";

import Box from "@/components/SharedComponents/Box";
import Badge from "@/components/Badge/";
import ButtonAsLink from "@/components/SharedComponents/Button/ButtonAsLink";
import Flex from "@/components/SharedComponents/Flex";
import { GridArea } from "@/components/Grid";
import Grid from "@/components/Grid/Grid";
import Illustration from "@/components/Illustration/Illustration";
import MaxWidth from "@/components/SharedComponents/MaxWidth";
import { Heading, LI, UL } from "@/components/Typography";
import Typography from "@/components/Typography/Typography";

const CurriculumTab: FC = () => {
  return (
    <Flex $background={"aqua"} $pv={24}>
      <MaxWidth $ph={[16]} $pb={24}>
        <Grid $cg={16}>
          <GridArea $colSpan={[12, 6]}>
            <Flex
              $flexDirection={"column"}
              $maxWidth={[640]}
              $pt={32}
              $alignItems={"flex-start"}
              $gap={24}
              $flex={"0 1 auto"}
            >
              <Heading $font={"heading-7"} tag={"h1"} $color={"grey70"}>
                Teachers & subject leads
              </Heading>
              <Heading $font={"heading-3"} tag={"h2"}>
                New curriculum plans
              </Heading>
              <Flex $flexDirection={"column"}>
                {" "}
                <Typography $font={"body-1"}>
                  Explore our new curriculum plans and early-release units.
                </Typography>
                <UL $mt={16} $font={"body-1"}>
                  <LI>National curriculum-aligned</LI>
                  <LI>Primary and secondary sequences</LI>
                  <LI>Designed by curriculum experts</LI>
                </UL>
              </Flex>

              <Flex $gap={24} $flexWrap={"wrap"} $pb={24}>
                <ButtonAsLink
                  label={"View curriculum plans"}
                  variant={"brushNav"}
                  page={"curriculum-landing-page"}
                  icon={"arrow-right"}
                  $iconPosition={"trailing"}
                  iconBackground="black"
                />
                <ButtonAsLink
                  label={"Early-release units"}
                  variant={"brushNav"}
                  page={"early-release-units-page"}
                  icon={"arrow-right"}
                  $iconPosition={"trailing"}
                  iconBackground="black"
                />
              </Flex>
            </Flex>
          </GridArea>
          <GridArea $colSpan={[12, 6]} $alignItems={"flex-end"}>
            <Flex
              $pv={64}
              $flexDirection={"column"}
              $justifyContent={"space-between"}
              $alignItems={"flex-end"}
              $flex={"0 1 auto"}
              $position={"relative"}
              $minWidth={[0, 350]}
              $display={["none", "flex"]}
              $maxWidth={524}
              $pl={20}
            >
              <Flex $flexDirection={"column"} $gap={10}>
                <Illustration
                  slug={"teacher-whiteboard"}
                  noCrop
                  $objectFit="contain"
                  priority
                  $ba={3}
                  $borderStyle={"solid"}
                  $borderColor={"black"}
                />
                <Typography $font={"body-1"} $color={"black"}>
                  50% of teachers feel more confident in curriculum design.
                </Typography>
              </Flex>

              <Box
                $position={"absolute"}
                $bottom={120}
                $left={-20}
                aria-hidden={true}
              >
                {" "}
                <Badge icon="bell" />
              </Box>
            </Flex>
          </GridArea>
        </Grid>
      </MaxWidth>
    </Flex>
  );
};

export default CurriculumTab;
