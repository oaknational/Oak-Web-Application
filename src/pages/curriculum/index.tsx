import { GetStaticProps, NextPage } from "next";

import { DEFAULT_SEO_PROPS } from "@/browser-lib/seo/Seo";
import AppLayout from "@/components/SharedComponents/AppLayout";
import getPageProps from "@/node-lib/getPageProps";
import HomePageTabImageNav from "@/components/GenericPagesComponents/HomePageTabImageNav";
import Banners from "@/components/CurriculumComponents/Banners";
import { HomePageLowerView } from "@/components/GenericPagesViews/HomePageLower/HomePageLower.view";
import CurriculumTab from "@/components/GenericPagesComponents/CurriculumTab";
import {
  getPropsFunction,
  HomePageProps,
} from "@/pages-helpers/home/getBlogPosts";

const Curriculum: NextPage<HomePageProps> = (props) => (
  <AppLayout seoProps={DEFAULT_SEO_PROPS} $background={"white"}>
    <Banners />
    <HomePageTabImageNav current={"curriculum"} />
    <CurriculumTab aria-current="page" />
    <HomePageLowerView posts={props.posts} />
  </AppLayout>
);

export const getStaticProps: GetStaticProps<HomePageProps> = async (
  context,
) => {
  return getPageProps({
    page: "curriculum-home-page::getStaticProps",
    context,
    getProps: getPropsFunction(context),
  });
};

export default Curriculum;
