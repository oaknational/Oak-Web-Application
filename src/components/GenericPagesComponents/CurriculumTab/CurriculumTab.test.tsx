import { act } from "@testing-library/react";

import CurriculumTab from "./CurriculumTab";

import { subjectPhaseOptions } from "@/utils/curriculum/fixtures";
import renderWithProviders from "@/__tests__/__helpers__/renderWithProviders";

window.matchMedia = () => ({ matches: true }) as unknown as MediaQueryList;

const render = renderWithProviders();

describe("CurriculumTab", () => {
  test("renders", () => {
    const { baseElement } = render(
      <CurriculumTab
        subjectPhaseOptions={{
          subjects: subjectPhaseOptions.options,
        }}
      />,
    );
    expect(baseElement).toMatchSnapshot();
  });

  // Note we don't have to do a full interactivity test here
  // as the <SubjectPhasePicker/> is tested elsewhere.
  test("interactive", () => {
    const { getAllByTestId, getByTestId } = render(
      <CurriculumTab
        subjectPhaseOptions={{
          subjects: subjectPhaseOptions.options,
        }}
      />,
    );

    act(() => {
      getAllByTestId("subject-picker-button")[0]?.click();
    });
    expect(getByTestId("close-modal-button"));
  });
});
