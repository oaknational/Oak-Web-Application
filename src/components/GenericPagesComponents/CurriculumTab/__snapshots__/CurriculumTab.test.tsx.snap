// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CurriculumTab renders 1`] = `
<body>
  <div>
    <div
      data-overlay-container="true"
    >
      <div
        class="sc-fqkvVR iKBTzE"
      >
        <div
          class="sc-fqkvVR sc-gsFSXq sc-cwHptR itRfeb ibSZJr"
        >
          <div
            class="sc-fqkvVR sc-gsFSXq fgBWrW hDihdg"
          >
            <div
              class="sc-fqkvVR sc-gsFSXq blCXVh eqRNuD"
            >
              <h1
                class="sc-dhKdcB cMkAuQ"
              >
                Teachers & subject leads
              </h1>
              <h2
                class="sc-dhKdcB bzyrBu"
              >
                Curriculum plans
              </h2>
              <div
                class="sc-fqkvVR sc-gsFSXq fgBWrW bOTBNE"
              >
                 
                <div
                  class="sc-fqkvVR sc-eldPxv iGOzG gvClhl"
                >
                  All of our curriculum plans are:
                </div>
                <ul
                  class="sc-fPXMVe lpuqhW"
                >
                  <li
                    class="sc-dAlyuH kPmtoE"
                  >
                    National curriculum-aligned
                  </li>
                  <li
                    class="sc-dAlyuH kPmtoE"
                  >
                    Sequenced across year groups
                  </li>
                  <li
                    class="sc-dAlyuH kPmtoE"
                  >
                    Designed by curriculum experts
                  </li>
                </ul>
              </div>
              <div
                class="sc-fqkvVR sc-gsFSXq bmnVAU erhMjA"
              >
                <a
                  class="sc-JrDLc cEyXEU"
                  href="/teachers/curriculum"
                >
                  <span
                    class="sc-kpDqfm psnGs"
                  >
                    <div
                      class="sc-fqkvVR sc-eldPxv dWmZcx iGHzbF"
                    >
                      Our curriculum planning approach
                    </div>
                  </span>
                  <div
                    class="sc-fqkvVR kbnxKi sc-fXSgeo"
                  >
                    <img
                      alt="chevron-right"
                      class="sc-dcJsrY cDKKvZ"
                      data-nimg="fill"
                      decoding="async"
                      loading="lazy"
                      src="https://next_public_oak_assets_host/NEXT_PUBLIC_OAK_ASSETS_PATH/v1707752509/icons/vk9xxxhnsltsickom6q9.svg"
                      style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
                    />
                  </div>
                </a>
              </div>
              <div
                class="sc-fqkvVR hEDAbB"
              >
                <div
                  aria-labelledby="choose-curriculum-label"
                  class="sc-fqkvVR fgBWrW"
                  role="group"
                >
                  <div
                    class="sc-fqkvVR iKkOLq"
                    id="choose-curriculum-label"
                  >
                    Choose a curriculum
                  </div>
                  <div
                    class="sc-fqkvVR czELrn"
                    data-testid="lot-picker"
                  >
                    <div
                      class="sc-fqkvVR sc-gsFSXq iRyopa juXCUD"
                    >
                      <div
                        class="sc-fqkvVR sc-gsFSXq gJgCTZ ectRTh"
                      >
                        <div
                          class="sc-fqkvVR sc-gsFSXq fxkSvJ kHulgy"
                          style="width: 50%;"
                        >
                          <div
                            class="sc-fqkvVR sc-9dfc1bad-0 sc-bdacaec4-2 jEHEPk fAkCpO hPodqY"
                          >
                            <button
                              class="sc-bdacaec4-1 cgYPZt"
                              data-testid="subject-picker-button"
                              title="Subject"
                            >
                              <div
                                class="sc-fqkvVR iHZIMJ"
                              >
                                <span
                                  class="sc-kpDqfm gGqndO"
                                  data-testid="subject-picker-button-heading"
                                >
                                  Subject
                                </span>
                                <p
                                  class="sc-dLMFU dULECm"
                                >
                                  Select
                                </p>
                              </div>
                            </button>
                          </div>
                        </div>
                        <div
                          class="sc-fqkvVR eIwgYx"
                        >
                          <div
                            aria-hidden="true"
                            class="sc-fqkvVR fgBWrW"
                            data-testid="brush-borders"
                          >
                            <svg
                              aria-hidden="true"
                              class="sc-df6cb28a-0 glfkwM"
                              height="100%"
                              name="box-border-left"
                              style="width: 3px;"
                              width="100%"
                              xmlns="http://www.w3.org/2000/svg"
                            >
                              <svg
                                fill="none"
                                preserveAspectRatio="none"
                                viewBox="0 0 2.72 359.41"
                                xmlns="http://www.w3.org/2000/svg"
                              >
                                <path
                                  d="M2.56 26.03c-.14-7.28.05-12.58 0-18.65C2.48 3.63 2.28.24 2.51 0H.02c.13 3.41 1.04 10.2.02 14.63-.1 9.52.02 311.42-.03 344.78h2.5c-.87-4.14.68-328.77.05-333.38Z"
                                  fill="currentColor"
                                />
                                <path
                                  d="M1.78 73.31c-.48-1.02.43-5.62.43-4.84.01.69.38 4.88-.43 4.84ZM1.56 142.12c1.14.07.13.67-.03.65-.47-.05-.76-.69.03-.65ZM1.65 196.21c.23.4.13 2.4.07 2.59-.36-1.73-.51-3.36-.07-2.59ZM2.48 27.51c0 .49 0 1.87-.11 3.12-.37 6.22-.42-1.85.11-3.12ZM2.01 65.86c.39.51.58 1.99-.23 1.95-.48-1.02-.25-2.56.23-1.95ZM1.02 132.42c-.47-.05-.49-1.39-.04-1.78.85-.76.2 1.8.04 1.78ZM.79 5.27c.07-1.81.65 1.18.59 5.76C1.27 17.8.69 6.25.79 5.27ZM.33 47.73c-.33-1.44.48-7.29.21-2.86-.12.57-.13 5.57-.21 2.86ZM.96 94.76c.28.95-.15 5.38-.31 5.36-.47-.05.08-6.12.31-5.36ZM1.23 100.12c-.07.9-.41 1.95-.58 1.95-.47-.05.64-2.73.58-1.95ZM1.38 144.69c.4-1.07-.4 9.48-.56 9.46-.47-.05.12-8.31.56-9.46ZM1.24 197.97c.23.4.13 2.4.07 2.59-.36-1.73-.51-3.36-.07-2.59ZM1.81 209.98c-.36-1.74-.84-7.52-.52-8.14l.5-1.57c.25.43.05 9.87.02 9.71ZM1.85 158.2c-.47-.05-.11-4.72.13-3.97.33 1.09.03 3.99-.13 3.97Z"
                                  fill="currentColor"
                                />
                              </svg>
                            </svg>
                          </div>
                        </div>
                        <div
                          class="sc-ad0ad6f5-0 jWQsSO"
                        >
                          <div
                            class="sc-fqkvVR sc-gsFSXq fxkSvJ ehZBcc"
                          >
                            <div
                              class="sc-fqkvVR sc-9dfc1bad-0 sc-bdacaec4-2 eMDDOo fAkCpO hPodqY"
                            >
                              <button
                                class="sc-bdacaec4-1 cgYPZt"
                                data-testid="phase-picker-button"
                                title="Phase"
                              >
                                <div
                                  class="sc-fqkvVR bBVkkf"
                                >
                                  <span
                                    class="sc-kpDqfm gGqndO"
                                    data-testid="phase-picker-button-heading"
                                  >
                                    School phase
                                  </span>
                                  <div
                                    class="sc-fqkvVR iiKnvq"
                                  >
                                    Select
                                  </div>
                                </div>
                              </button>
                            </div>
                            <div
                              class="sc-fqkvVR sc-gsFSXq edMMHU bKBnOm"
                            >
                              <div
                                class="sc-fqkvVR sc-iHbSHJ eJBGXR fIxqXk"
                              >
                                <div
                                  class="sc-fqkvVR iPpXCP grey-shadow"
                                />
                                <div
                                  class="sc-fqkvVR iPpXCP yellow-shadow"
                                />
                                <button
                                  class="sc-lcIPJg sc-hRJfrW lcdeJA eJSoyq internal-button"
                                  data-testid="lot-picker-view-curriculum-button"
                                >
                                  <div
                                    class="sc-fqkvVR sc-gsFSXq fgBWrW dzOlKm"
                                  >
                                    <span
                                      class="sc-kpDqfm jpmbqG"
                                    >
                                      View
                                    </span>
                                    <div
                                      class="sc-fqkvVR kbnxKi"
                                    >
                                      <img
                                        alt="arrow-right"
                                        class="sc-dcJsrY bDcxkE"
                                        data-nimg="fill"
                                        decoding="async"
                                        loading="lazy"
                                        src="https://next_public_oak_assets_host/NEXT_PUBLIC_OAK_ASSETS_PATH/v1707149070/icons/fv0z57zerrioft52dd9n.svg"
                                        style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
                                      />
                                    </div>
                                  </div>
                                </button>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="sc-fqkvVR sc-gsFSXq ivrxlG gdRCvt"
            >
              <div
                class="sc-fqkvVR sc-gsFSXq cVTiFC imOBLi"
              >
                <img
                  alt=""
                  class="sc-6170517-0 lnqwbG"
                  data-nimg="1"
                  decoding="async"
                  fetchpriority="high"
                  height="628"
                  src="https://NEXT_PUBLIC_SANITY_ASSET_CDN_HOST/images/NEXT_PUBLIC_SANITY_PROJECT_ID/NEXT_PUBLIC_SANITY_DATASET/b63430479cc683ffce99280e132c671325d1a0fd-1048x628.jpg?w=1080&h=647&fm=webp&q=80&fit=clip&auto=format"
                  srcset="https://NEXT_PUBLIC_SANITY_ASSET_CDN_HOST/images/NEXT_PUBLIC_SANITY_PROJECT_ID/NEXT_PUBLIC_SANITY_DATASET/b63430479cc683ffce99280e132c671325d1a0fd-1048x628.jpg?w=640&h=383&fm=webp&q=80&fit=clip&auto=format 1x, https://NEXT_PUBLIC_SANITY_ASSET_CDN_HOST/images/NEXT_PUBLIC_SANITY_PROJECT_ID/NEXT_PUBLIC_SANITY_DATASET/b63430479cc683ffce99280e132c671325d1a0fd-1048x628.jpg?w=1080&h=647&fm=webp&q=80&fit=clip&auto=format 2x"
                  style="color: transparent;"
                  width="520"
                />
                <div
                  class="sc-a9830b5a-0 jCuiZA"
                >
                  <div
                    class="sc-fqkvVR bHatqy"
                  >
                    <img
                      alt="looping-line-5"
                      class="sc-dcJsrY bDcxkE"
                      data-nimg="fill"
                      decoding="async"
                      loading="lazy"
                      src="https://next_public_oak_assets_host/NEXT_PUBLIC_OAK_ASSETS_PATH/v1740665310/OWA/ui-graphics/looping-line-5_vdknco.svg"
                      style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
                    />
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="sc-fqkvVR gEooAa"
          >
            <div
              aria-labelledby="choose-curriculum-label"
              class="sc-fqkvVR fgBWrW"
              role="group"
            >
              <div
                class="sc-fqkvVR iKkOLq"
                id="choose-curriculum-label"
              >
                Choose a curriculum
              </div>
              <div
                class="sc-fqkvVR czELrn"
                data-testid="lot-picker"
              >
                <div
                  class="sc-fqkvVR sc-gsFSXq iRyopa juXCUD"
                >
                  <div
                    class="sc-fqkvVR sc-gsFSXq gJgCTZ ectRTh"
                  >
                    <div
                      class="sc-fqkvVR sc-gsFSXq fxkSvJ kHulgy"
                      style="width: 50%;"
                    >
                      <div
                        class="sc-fqkvVR sc-9dfc1bad-0 sc-bdacaec4-2 jEHEPk fAkCpO hPodqY"
                      >
                        <button
                          class="sc-bdacaec4-1 cgYPZt"
                          data-testid="subject-picker-button"
                          title="Subject"
                        >
                          <div
                            class="sc-fqkvVR iHZIMJ"
                          >
                            <span
                              class="sc-kpDqfm gGqndO"
                              data-testid="subject-picker-button-heading"
                            >
                              Subject
                            </span>
                            <p
                              class="sc-dLMFU dULECm"
                            >
                              Select
                            </p>
                          </div>
                        </button>
                      </div>
                    </div>
                    <div
                      class="sc-fqkvVR eIwgYx"
                    >
                      <div
                        aria-hidden="true"
                        class="sc-fqkvVR fgBWrW"
                        data-testid="brush-borders"
                      >
                        <svg
                          aria-hidden="true"
                          class="sc-df6cb28a-0 glfkwM"
                          height="100%"
                          name="box-border-left"
                          style="width: 3px;"
                          width="100%"
                          xmlns="http://www.w3.org/2000/svg"
                        >
                          <svg
                            fill="none"
                            preserveAspectRatio="none"
                            viewBox="0 0 2.72 359.41"
                            xmlns="http://www.w3.org/2000/svg"
                          >
                            <path
                              d="M2.56 26.03c-.14-7.28.05-12.58 0-18.65C2.48 3.63 2.28.24 2.51 0H.02c.13 3.41 1.04 10.2.02 14.63-.1 9.52.02 311.42-.03 344.78h2.5c-.87-4.14.68-328.77.05-333.38Z"
                              fill="currentColor"
                            />
                            <path
                              d="M1.78 73.31c-.48-1.02.43-5.62.43-4.84.01.69.38 4.88-.43 4.84ZM1.56 142.12c1.14.07.13.67-.03.65-.47-.05-.76-.69.03-.65ZM1.65 196.21c.23.4.13 2.4.07 2.59-.36-1.73-.51-3.36-.07-2.59ZM2.48 27.51c0 .49 0 1.87-.11 3.12-.37 6.22-.42-1.85.11-3.12ZM2.01 65.86c.39.51.58 1.99-.23 1.95-.48-1.02-.25-2.56.23-1.95ZM1.02 132.42c-.47-.05-.49-1.39-.04-1.78.85-.76.2 1.8.04 1.78ZM.79 5.27c.07-1.81.65 1.18.59 5.76C1.27 17.8.69 6.25.79 5.27ZM.33 47.73c-.33-1.44.48-7.29.21-2.86-.12.57-.13 5.57-.21 2.86ZM.96 94.76c.28.95-.15 5.38-.31 5.36-.47-.05.08-6.12.31-5.36ZM1.23 100.12c-.07.9-.41 1.95-.58 1.95-.47-.05.64-2.73.58-1.95ZM1.38 144.69c.4-1.07-.4 9.48-.56 9.46-.47-.05.12-8.31.56-9.46ZM1.24 197.97c.23.4.13 2.4.07 2.59-.36-1.73-.51-3.36-.07-2.59ZM1.81 209.98c-.36-1.74-.84-7.52-.52-8.14l.5-1.57c.25.43.05 9.87.02 9.71ZM1.85 158.2c-.47-.05-.11-4.72.13-3.97.33 1.09.03 3.99-.13 3.97Z"
                              fill="currentColor"
                            />
                          </svg>
                        </svg>
                      </div>
                    </div>
                    <div
                      class="sc-ad0ad6f5-0 jWQsSO"
                    >
                      <div
                        class="sc-fqkvVR sc-gsFSXq fxkSvJ ehZBcc"
                      >
                        <div
                          class="sc-fqkvVR sc-9dfc1bad-0 sc-bdacaec4-2 eMDDOo fAkCpO hPodqY"
                        >
                          <button
                            class="sc-bdacaec4-1 cgYPZt"
                            data-testid="phase-picker-button"
                            title="Phase"
                          >
                            <div
                              class="sc-fqkvVR bBVkkf"
                            >
                              <span
                                class="sc-kpDqfm gGqndO"
                                data-testid="phase-picker-button-heading"
                              >
                                School phase
                              </span>
                              <div
                                class="sc-fqkvVR iiKnvq"
                              >
                                Select
                              </div>
                            </div>
                          </button>
                        </div>
                        <div
                          class="sc-fqkvVR sc-gsFSXq edMMHU bKBnOm"
                        >
                          <div
                            class="sc-fqkvVR sc-iHbSHJ eJBGXR fIxqXk"
                          >
                            <div
                              class="sc-fqkvVR iPpXCP grey-shadow"
                            />
                            <div
                              class="sc-fqkvVR iPpXCP yellow-shadow"
                            />
                            <button
                              class="sc-lcIPJg sc-hRJfrW lcdeJA eJSoyq internal-button"
                              data-testid="lot-picker-view-curriculum-button"
                            >
                              <div
                                class="sc-fqkvVR sc-gsFSXq fgBWrW dzOlKm"
                              >
                                <span
                                  class="sc-kpDqfm jpmbqG"
                                >
                                  View
                                </span>
                                <div
                                  class="sc-fqkvVR kbnxKi"
                                >
                                  <img
                                    alt="arrow-right"
                                    class="sc-dcJsrY bDcxkE"
                                    data-nimg="fill"
                                    decoding="async"
                                    loading="lazy"
                                    src="https://next_public_oak_assets_host/NEXT_PUBLIC_OAK_ASSETS_PATH/v1707149070/icons/fv0z57zerrioft52dd9n.svg"
                                    style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
                                  />
                                </div>
                              </div>
                            </button>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="sc-fqkvVR grAKRD"
      data-testid="cookie-banner"
    >
      <div
        class="sc-fqkvVR jMNzyV"
      >
        <div
          class="sc-fqkvVR sc-gsFSXq krbSaB iLFXKS"
        >
          <div
            class="sc-fqkvVR kJPpVN"
          >
            <strong
              class="sc-kpDqfm bwvRQn"
            >
              This site uses cookies to store information on your computer.
            </strong>
             
            Some of these cookies are essential, while others help us to improve your experience by providing insights into how the site is being used.
          </div>
          <div
            class="sc-fqkvVR sc-gsFSXq fgBWrW ftCnYI"
          >
            <div
              class="sc-fqkvVR sc-gsFSXq fOyTgd gZFQMg"
            >
              <button
                class="sc-JrDLc cEyXEU"
                data-testid="cookie-banner-reject"
                type="button"
              >
                <span
                  class="sc-kpDqfm psnGs"
                >
                  Reject non-essential cookies
                </span>
              </button>
            </div>
            <div
              class="sc-fqkvVR sc-iHbSHJ eJBGXR fIxqXk"
            >
              <div
                class="sc-fqkvVR iPpXCP grey-shadow"
              />
              <div
                class="sc-fqkvVR iPpXCP yellow-shadow"
              />
              <button
                class="sc-lcIPJg sc-hRJfrW kjNddQ dpmFSv internal-button"
              >
                <div
                  class="sc-fqkvVR sc-gsFSXq fgBWrW dzOlKm"
                >
                  <span
                    class="sc-kpDqfm jpmbqG"
                  >
                    Cookie settings
                  </span>
                </div>
              </button>
            </div>
            <div
              class="sc-fqkvVR sc-iHbSHJ eJBGXR fIxqXk"
            >
              <div
                class="sc-fqkvVR iPpXCP grey-shadow"
              />
              <div
                class="sc-fqkvVR iPpXCP yellow-shadow"
              />
              <button
                class="sc-lcIPJg sc-hRJfrW lcdeJA eJSoyq internal-button"
                data-testid="cookie-banner-accept"
              >
                <div
                  class="sc-fqkvVR sc-gsFSXq fgBWrW dzOlKm"
                >
                  <span
                    class="sc-kpDqfm jpmbqG"
                  >
                    Accept all cookies
                  </span>
                </div>
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</body>
`;
